src/test/java/com/epam/tc/hw1/DivisionCalculatorTest.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+
+public class DivisionCalculatorTest extends TestingCalculatorOperations {
+
+    @DataProvider(name = "DPDivision")

```

Comment:
It will be more suitable to use the DataProviderDivision name for Data Provider

Reply:
Ok, thanks.